keras_core/utils/dataset_utils_test.py

```diff
@@ -0,0 +1,151 @@
+from keras_core.testing import test_case
+from keras_core.utils import naming
+from keras_core.utils.module_utils import tensorflow as tf
+from keras_core.utils.dataset_utils import split_dataset
+import numpy as np
+
+class DatasetUtilsTest(test_case.TestCase):
+    def test_split_dataset_list(self):
+        n_sample, n_cols, n_pred, left_size, right_size = 100, 2, 1, 0.2, 0.8
```

Comment:
Make sure to run `sh shell/format.sh` and keep lines under 80 chars

Reply:
Done